cross-dom-bridge-eth/index.js

```diff
@@ -0,0 +1,128 @@
+#! /usr/local/bin/node
+
+// Transfers between L1 and L2 using the Optimism SDK
+
+const ethers = require("ethers")
+const optimismSDK = require("@eth-optimism/sdk")
+require('dotenv').config()
+
+
+const mnemonic = process.env.MNEMONIC
+const l1Url = `https://eth-goerli.g.alchemy.com/v2/${process.env.GOERLI_ALCHEMY_KEY}`
+const l2Url = `https://opt-goerli.g.alchemy.com/v2/${process.env.OPTIMISM_GOERLI_ALCHEMY_KEY}`
+
+
+// Global variable because we need them almost everywhere
+let crossChainMessenger
+let addr    // Our address
+
+const getSigners = async () => {
+    const l1RpcProvider = new ethers.providers.JsonRpcProvider(l1Url)
+    const l2RpcProvider = new ethers.providers.JsonRpcProvider(l2Url)
+    const hdNode = ethers.utils.HDNode.fromMnemonic(mnemonic)
+    const privateKey = hdNode.derivePath(ethers.utils.defaultPath).privateKey
+    const l1Wallet = new ethers.Wallet(privateKey, l1RpcProvider)
+    const l2Wallet = new ethers.Wallet(privateKey, l2RpcProvider)
+
+    return [l1Wallet, l2Wallet]
+}   // getSigners
+
+
+const setup = async() => {
+  const [l1Signer, l2Signer] = await getSigners()
+  addr = l1Signer.address
+  crossChainMessenger = new optimismSDK.CrossChainMessenger({
+      l1ChainId: 5,    // Goerli value, 1 for mainnet
+      l2ChainId: 420,  // Goerli value, 10 for mainnet
+      l1SignerOrProvider: l1Signer,
+      l2SignerOrProvider: l2Signer
+  })
+}    // setup
+
+
+
+const gwei = 1000000000n
```

Comment:
Yea I am really not a fan of the `n` annotation.

Reply:
You prefer `BigInt(1e9)`? Sure, I can do that.